rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_reader :person, :book
```

Comment:
- [ ] @momo-87 kindly move the person and book to attr_accessor, this is because we not only reading them but also changing the variables' content.

Reply:
Hi @danmainah ,
I really appreciate the time and effort you took in reviewing my code. Thank you.
For this point, I made the changes by defining book and person setter in the class Rental. So to avoid duplication, I kept  person and book to attr_reader at the top part of the class.

Thank you for your suggestion.